python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -1166,3 +1172,15 @@ def print_other_sessions(self):
         if len(other_sessions) > 0:
             logger.warning("%i other concurrent sessions:\n * %s" % (len(other_sessions), "\n * ".join(other_sessions)))
 
+    def has_loaded(self, obj):
+
+        try:
+            index = self.find(obj)
+        except:
+            return False
```

Comment:
Lots of functions can use doc strings, I'll get around to this as a general point in the near future.
The error here needs to be better defined in the ```find``` method I suspect. I'll check there and update this.